cogs/induct.py

```diff
@@ -248,9 +257,19 @@ async def _perform_induction(self, ctx: TeXBotApplicationContext, induction_memb
             recent_message: discord.Message
             for recent_message in await intro_channel.history(limit=30).flatten():
                 if recent_message.author.id == induction_member.id:
-                    if tex_emoji:
-                        await recent_message.add_reaction(tex_emoji)
-                    await recent_message.add_reaction("👋")
+                    try:
+                        if tex_emoji:
+                            await recent_message.add_reaction(tex_emoji)
+                        await recent_message.add_reaction("👋")
+                    except discord.Forbidden as forbidden_error:
+                        if "Reaction blocked" in forbidden_error.text:
```

Comment:
Making the check lowercase makes it more resiliant to API updates:

```python
forbidden_error: discord.Forbidden
try:
    ...
except discord.Forbidden as forbidden_error:
    REACTION_WAS_BLOCKED: bool = bool(
        "reaction" in forbidden_error.text.lower()
        and "blocked" in forbidden_error.text.lower()
    )
    if REACTION_WAS_BLOCKED:
        ...
```

Reply:
what's the point in the bool block here? can it not just be an if?

Reply:
The line will be too long if `"reaction" not in forbidden_error.text.lower() or "blocked" not in forbidden_error.text.lower()` is used in the `if` statement.

Reply:
Why do they need to be separate checks? Can it not just be "reaction blocked" blah blah blah? having them as separate checks could catch other errors introduced in future that use one of those words but refer to something else - while that's still possible using the whole phrase it's much less likely

Reply:
Yeah there's an argument to be made either way. Splitting it up would allow for changes of the message to `reaction was blocked`, but would also incorrectly accept it if the message changed to `deletion blocked, and there was a reaction`.

Feel free to leave it as `if "reaction blocked" not in forbidden_error.text.lower():`

Reply:
solved by just searching for the error code instead